packages/core/src/external/fhir/adt-encounters.ts

```diff
@@ -8,6 +9,18 @@ import { mergeBundles } from "./bundle/utils";
 
 const s3Utils = new S3Utils(Config.getAWSRegion());
 
+export function createCxIdPtIdPrefix({ cxId, patientId }: { cxId: string; patientId: string }) {
```

Comment:
I've been following the [naming convention](https://www.notion.so/metriport/Naming-convention-217b51bb9040804a8f27fe70e160d395?source=copy_link#217b51bb904080a2bf3ccc8cfbe93acb) using "buildXyz", but I honestly don't care either way since `createXYZ` also makes sense.